### PR TITLE
[JOLINT-50] info text on chapter overview

### PR DESCRIPTION
**Pull request by idamakela**

### CODE REVIEW

File: app/consent-form/_components/InfoText.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const InfoTextContent: string =
```
Comment (ell-ska) [inline app/consent-form/_components/InfoText.tsx:1]: i would pass this down as a prop if you want to separate it from the rest of the component, maybe with this text as a default value

File: app/consent-form/page.tsx [hunk 2]
```diff
@@ -8,6 +8,7 @@ import { cn } from '@/utils/classnames'
 import Title from '@/app/consent-form/_components/Title'
 import FormButtons from '@/app/consent-form/_components/form/FormButtons'
 import type { ChapterKeys } from '@/utils/types'
+import InfoText from '@/app/consent-form/_components/InfoText'
```
Comment (ell-ska) [inline app/consent-form/page.tsx:11]: i would move this up to the rest of the components

**Review by ell-ska — CHANGES_REQUESTED** [review]

File: app/consent-form/_components/InfoText.tsx (hunk 1 above)
Comment (idamakela) [reply app/consent-form/_components/InfoText.tsx:1]: OOoo i like that idea

**Review by ell-ska — APPROVED** [review]

**Comment by idamakela** [comment]
### Ready for dev test!

**Comment by DavidDyberg** [comment]
Dev test.
Looks good! 
Ready for production

**Pull request merged**